src/actors/client/ActorClient/ActorClientGRPC.ts

```diff
@@ -0,0 +1,278 @@
+import { Empty } from "google-protobuf/google/protobuf/empty_pb";
+import { Any } from "google-protobuf/google/protobuf/any_pb";
+import { ExecuteActorStateTransactionRequest, GetActorStateRequest, GetActorStateResponse, GetMetadataResponse, InvokeActorRequest, InvokeActorResponse, RegisterActorReminderRequest, RegisterActorTimerRequest, TransactionalActorStateOperation, UnregisterActorReminderRequest, UnregisterActorTimerRequest } from '../../../proto/dapr/proto/runtime/v1/dapr_pb';
+import GRPCClient from '../../../implementation/Client/GRPCClient/GRPCClient';
+import { OperationType } from '../../../types/Operation.type';
+import { ActorReminderType } from '../../../types/ActorReminder.type';
+import { ActorTimerType } from '../../../types/ActorTimer.type';
+import IClientActor from '../../../interfaces/Client/IClientActor';
+import { KeyValueType } from '../../../types/KeyValue.type';
+
+// https://docs.dapr.io/reference/api/actors_api/
+export default class ActorClientGRPC implements IClientActor {
+  client: GRPCClient;
+
+  constructor(client: GRPCClient) {
+    this.client = client;
+  }
+
+  async invoke(method: "GET" | "POST" | "PUT" | "DELETE", actorType: string, actorId: string, methodName: string, body?: any): Promise<object> {
```

Comment:
method seems to be unused, which makes sense because it is always Post.

Reply:
I do agree that it might be easier to have default POST and move method to the end as optional

Reply:
Please, keep POST or PUT only. No need to expose this. Here is why:

Java does POST only: https://github.com/dapr/java-sdk/blob/master/sdk-actors/src/main/java/io/dapr/actors/client/DaprHttpClient.java
.Net is using PUT only: https://github.com/dapr/dotnet-sdk/blob/0c9d6a45c8d3792a92d7141056c390bea098d02b/src/Dapr.Actors/DaprHttpInteractor.cs#L199

Runtime does indeed accept all these HTTP Methods: https://github.com/dapr/dapr/blob/ab1c6a0ac7041ff9da34626e05b3628277f61acb/pkg/http/api.go#L297
On the other hand, when invoking actor locally, we only use PUT. So, the method you use in the SDK does not matter in the end. This also allows the gRPC API to work seamlessly.

Reply:
Ok, I will remove the method